index.d.ts

```diff
@@ -0,0 +1,30 @@
+import EventEmitter from 'bare-events'
+
+declare class Signal extends EventEmitter<{ close: []; signal: [number] }> {
```

Comment:
```suggestion
declare class Signal extends EventEmitter<{ close: []; signal: [signal: number] }> {
```

TypeScript uses the tuple label as the argument name in autocompletion. For lack of a label, it will generate one of the form `arg_<n>`.